stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -18,6 +19,14 @@ from = (start, stop=now(), fn) =>
 // Log records notification events
 log = (tables=<-) => tables |> to(bucket: bucket)
```

Comment:
This should use `experimental.to` correct? Since the data is in pivoted form?

Reply:
Yes, that is likely the case. I'll update this.